requirements-test.txt

```diff
@@ -7,3 +7,5 @@ pytest-flakes==1.0.1
 pytest-mccabe==0.1
 pytest-pep8==1.0.6
 responses==0.5.0
+
+-e git+https://github.com/asher/chronos-python.git@e531f47d35d1d7b08d8e58abd58c3767af13f584#egg=chronos-python
```

Comment:
This answer shows what you'll need to do: http://stackoverflow.com/questions/32688688/how-to-write-setup-py-to-include-a-git-repo-as-a-dependency (more or less)